samples: wifi: p2p: Update README (#28163)
nrfconnect/sdk-nrf: samples/wifi/p2p/README.rst

.. options-from-kconfig::
   :show-type:

> In CLI mode, it is mandatory to set the :kconfig:option:`CONFIG_SAMPLE_P2P_PEER_ADDRESS` Kconfig option to connect to a peer.

Reviewer: ```suggestion
In CLI mode, you must set the :kconfig:option:`CONFIG_SAMPLE_P2P_PEER_ADDRESS` Kconfig option to connect to a peer.
```
